docs/io/visualization/montecarlo_packet_visualization.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>I guess we don't care about adding docstrings for these functions since they'll be changed eventually, but you should add them in the full module PR for sure!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>yeah, I surely will.</p>